pippo-core/src/main/java/ro/pippo/core/Pippo.java

```diff
@@ -87,16 +87,12 @@ public WebServer getServer() {
      * Entry point for a custom WebServer.
      * The idea is to create a custom WebServer if you want to override some aspects (method) of that server or
      * if you want free access to the servlet container (Jetty, Tomcat, ...).
-     *
      * <p>
-     * Show below the code for a <@code>JettyServer</@code> with persistent sessions.
-     * </p>
-     *
+     * Show below the code for a <code>JettyServer</code> with persistent sessions.
      * <pre>
-     * <@code>
```

Comment:
https://stackoverflow.com/a/19302439

Reply:
this particular entry has incorrect `<@code>` syntax (instead of `${@code}`)